src/lib/form/validation.ts

```diff
@@ -18,7 +18,7 @@ export type Validator = (
 export const validateRequired: Validator = (
   value,
   field,
-  message = 'Field "{name}" is required',
+  message = 'Veld "{name}" is verplicht',
```

Comment:
So this can be a constant, optionally document form behaviour overriding the final label